themes/app/templates/Includes/HomePageTop.ss

```diff
@@ -0,0 +1,15 @@
+<div class="homepage__top">
+        <div class="homepage__banner row align-items-center">
+            <div class="homepage__banner-text lead text-md-left text-sm-center col-md-6 col-sm-12 col-12 order-md-1 order-sm-12 order-12">
```

Comment:
i think here you can remove `col-12` and `col-sm-12` as bootstrap should know how to split your layout with a single class of `col-md-6` :)